framework/di/NotInstantiableException.php

```diff
@@ -16,7 +17,7 @@
  * @author Sam Mousa <sam@mousa.nl>
  * @since 2.0.9
  */
-class NotInstantiableException extends InvalidConfigException
+class NotInstantiableException extends InvalidConfigException implements NotFoundExceptionInterface
```

Comment:
This does not seem to be appropriate: `NotFoundExceptionInterface` means there is NO definition of the entity inside the container, while `NotInstantiableException` means the definition is invalid, pointing to something, which can not be intantiated like interface or absract class.
These exeptions can not be made equal.

Reply:
The PSR does not have something for not instantiable.